src/integrationTest/java/uk/gov/hmcts/reform/translate/controllers/DictionaryControllerIT.java

```diff
@@ -73,10 +78,39 @@ void shouldThrowExceptionWhenDictionaryReturnsDuplicateEnglishPhrases() {
         @Test
         void shouldReturn403WhenUserDoesNotHaveManageTranslationsRole() throws Exception {
             stubUserInfo("unknown-role");
-            mockMvc.perform(get(URL)
+            mockMvc.perform(get(DICTIONARY_URL)
                                 .contentType(MediaType.APPLICATION_JSON_VALUE))
                 .andExpect(status().is(403))
                 .andReturn();
         }
     }
+
+    @Nested
+    class GetTranslations {
+        private static final String TRANSLATIONS_URL = "/translation/cy";
```

Comment:
as above regarding URL constants, if change is made repeate here.

Reply:
same as above

Reply:
Resolved in #42